local-running/src/main/java/uk/gov/di/ipv/coreback/sqs/SqsReader.java

```diff
@@ -27,7 +27,7 @@ public SqsReader(RequestHandler<SQSEvent, SQSBatchResponse> sqsHandler) {
         this.sqs =
                 SqsClient.builder()
                         .region(EU_WEST_2)
-                        .httpClientBuilder(UrlConnectionHttpClient.builder())
+                        .httpClientBuilder(AwsCrtHttpClient.builder())
```

Comment:
(@Wynndow we should probably check if this client actually works in local-running mode - I think it should...)